spec/controllers/target_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+require 'database_cleaner'
+
+DatabaseCleaner.strategy = :truncation
+
+describe TargetsController, type: controller do
+  before(:each) do
+    @target = create(:target)
```

Comment:
use `let(:target) { create :target }`  instead of instace variables within a test